perception/traffic_light_recognition/traffic_light_ssd_fine_detector/README.md

```diff
@@ -1,9 +1,63 @@
 # traffic_light_ssd_fine_detector
 
-This is a traffic light fine detector implementation of MobileNetV2 + SSDLite.
+## Purpose
+
+It is a package for traffic light detection using MobileNetV2 and SSDLite.
 
 The trained model is based on [pytorch-ssd](https://github.com/qfgaohao/pytorch-ssd).
 
-## Reference
+## Inner-workings / Algorithms
+
+Based on the camera image and the global ROI array detected by `map_based_detection` node, a CNN-based detection method enables highly accurate traffic light detection.
+
+### Cite
```

Comment:
Saito-san says:
<https://github.com/tier4/AutowareArchitectureProposal.iv/pull/604#issuecomment-968429019>
> 論文は
> 
> ```
> ## (Optional) References/External links
> ```
> 
> を追加して移しておいていただけると助かります

Reply:
Thanks! It was fixed by https://github.com/tier4/AutowareArchitectureProposal.iv/pull/637/commits/acd4a45c36c77b23a608606901127c35b17660e1